java/code/src/com/redhat/rhn/frontend/strings/java/StringResource_en_US.xml

```diff
@@ -8648,7 +8648,7 @@ Alternatively, you will want to download &lt;strong&gt;Incremental Channel Conte
       <trans-unit id="contentmanagement.validation.nomodulefilters" xml:space="preserve">
         <source>No module filters attached. Module metadata will be cloned to target.</source>
         <context-group name="ctx">
-          <context context-type="sourcefile">com.redhat.rhn.domain.contentmgmt.validation.ModularSourcesValidator</context>
+          <context context-type="sourcefile">com.redhat.hn.domain.contentmgmt.validation.ModularSourcesValidator</context>
```

Comment:
this seems to be wrong

Reply:
Hopalla. I made a mistake during conflict resolution. Thanks!